migrations/sql/R__view_bonds.sql

```diff
@@ -101,6 +101,35 @@ FROM
                   null as "update_user", 
                   'Yes' AS "is_current_record", 
                   'MMS_FDW' as "source"
-     FROM SES_BOND_STAGING) AS bond_data
+     FROM SES_BOND_STAGING
```

Comment:
On line 87 of this file we should be setting the closed date.

Reply:
Added logic to set closed date :)